portfolio.js

```diff
@@ -13,15 +15,26 @@ Project.prototype.toHtml = function() {
   $newProject.find('a').text(this.title);
   $newProject.attr('.date-created', this.dateCreated);
   $newProject.attr('src', this.image);
-  $newProject.attr('href', this.fileSource); //need to chain .html jQ method on?
+  $newProject.attr('.description', this.description);
```

Comment:
Generally, when we make a constructor, we only want it to do one thing - make object instances for us. Your code would be more modular and decoupled (one function doing one thing) if you were to extract your jQuery method calls into another function, like toHtml in our class code.